WordPress/UITestsFoundation/Screens/ReaderScreen.swift

```diff
@@ -104,27 +104,28 @@ public class ReaderScreen: ScreenObject {
         )
     }
 
-    public func openLastPost() -> ReaderScreen {
-        getLastPost().tap()
+    public func openLastPost() throws -> ReaderScreen {
+        try getLastPost().tap()
         return self
     }
 
-    public func openLastPostInSafari() -> ReaderScreen {
-        getLastPost().buttons["More"].tap()
+    public func openLastPostInSafari() throws -> ReaderScreen {
+        try getLastPost().buttons["More"].tap()
         visitButton.tap()
         return self
     }
 
     public func openLastPostComments() throws -> CommentsScreen {
-        let commentButton = getLastPost().buttons["Comment"]
-        guard commentButton.waitForIsHittable() else { fatalError("ReaderScreen.Post: Comments button not loaded") }
+        let commentButton = try getLastPost().buttons["Comment"]
+        guard commentButton.waitForIsHittable() else { throw UIElementNotFoundError(message: "ReaderScreen.Post: Comments button not loaded") }
         commentButton.tap()
         return try CommentsScreen()
     }
 
     @discardableResult
-    public func getLastPost() -> XCUIElement {
-        guard let post = app.cells.lastMatch else { fatalError("ReaderScreen: No posts loaded") }
+    public func getLastPost() throws -> XCUIElement {
+        // Crash from testViewPostInSafari
```

Comment:
Is this comment necessary?

Reply:
Ah good catch. That was a note to myself. Removed in https://github.com/wordpress-mobile/WordPress-iOS/pull/22738/commits/1f50f52bcb94f677ef779472c49bf58950f5e4bd 🙈